procrastinate/contrib/django/django_connector.py

```diff
@@ -127,7 +127,9 @@ def get_worker_connector(self) -> connector.BaseAsyncConnector:
         """
         alias = utils.get_setting("DATABASE_ALIAS", default="default")
 
-        if utils.package_is_installed("psycopg3"):
+        if utils.package_is_installed("psycopg") and utils.package_is_version(
+            "psycopg", 3
+        ):
```

Comment:
Let's assume that there won't be a v4 (or we'll have ample time to adapt this code by then)